pyresample/geometry.py

```diff
@@ -706,6 +739,7 @@ def __init__(self, lons, lats, nprocs=1):
             raise ValueError('lon and lat arrays must have same shape')
         elif lons.ndim > 2:
             raise ValueError('Only 1 and 2 dimensional swaths are allowed')
+        self.crs = CRS(proj='longlat')
```

Comment:
Love it! Could we be more specific though? I think most people would say EPSG:4326, but I think that has fallen out of favor due to it not describing/handling time in its definition. I've asked @snowman2 if he has ideas. Either way I think an explicit datum/ellps would be good.

Reply:
I think this is a reasonable default unless you have more information. There is the `CRS.geodetic_crs` property if you want to get a geographic CRS with he same datum & ellipse as another CRS. You can also build a CRS if you have more information: https://pyproj4.github.io/pyproj/stable/build_crs.html

Reply:
I added `WGS84` as default ellipsoid, can be passed to the constructor